server/src/main/scala/com/cloudera/livy/server/LivyServer.scala

```diff
@@ -122,6 +118,13 @@ class LivyServer extends Logging {
       server.context.addFilter(csrfHolder, "/*", EnumSet.allOf(classOf[DispatcherType]))
     }
 
+    if (livyConf.getBoolean(ACCESS_CONTROL_ENABLED)) {
+      info("Access control is enabled.")
```

Comment:
If `AUTH_TYPE` is not defined in the conf you should probably error out, or at least print a warning, since this won't do anything in that case.